.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,114 @@
+---
+name: Bug report
+description: Problems and issues with code or docs
+labels: ["kind:bug"]
+body:
+  - type: dropdown
+    attributes:
+      label: Apache Airflow version
+      description: >
+        What Apache Airflow version are you using? Only Airflow 2 is supported for bugs. If you wish to
+        discuss Airflow 1.10, open [Discussion](https://github.com/apache/airflow/discussions) instead!
+      multiple: false
+      options:
+        - "2.1.3 (latest)"
+        - "2.1.2"
+        - "2.1.1"
+        - "2.1.0"
+        - "2.0.2"
+        - "2.0.1"
+        - "2.0.0"
+        - "main"
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Operating System
+      description: What Operating System are you using?
+      placeholder: "You can get it via `cat /etc/os-release` for example"
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Versions of Apache Airflow Providers
+      description: What Apache Airflow Providers versions are you using?
+      placeholder: You can use `pip freeze | grep apache-airflow-providers` (you can leave only relevant ones)
+  - type: dropdown
+    attributes:
+      label: Deployment
+      description: What kind of deployment do you have?
+      multiple: false
+      options:
+        - "Managed Services (Astronomer, Composer, MWAA etc.)"
+        - "Official Apache Airflow Helm Chart"
+        - "Other 3rd-party Helm chart"
+        - "Docker-Compose"
+        - "Other Docker-based deployment"
+        - "Virtualenv installation"
+        - "Other"
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Deployment details
+      description: Additional description of your deployment.
+      placeholder: >
+        Enter any relevant details of your deployment. Especially version of your tools,
+        software (docker-compose, helm, k8s, etc.), any customisation and configuration you added.
+  - type: textarea
+    attributes:
+      label: What happened
+      description: Describe what happened.
+      placeholder: >
+        Please provide the context in which the problem occurred and explain what happened!
```

```suggestion
        Please provide the context in which the problem occurred and explain what happened
```
nit - feeling the exclamation mark is a little too much here 😁